### DISCUSSION TITLE
ABC Kids! [AU]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#898] [Q&A]
### Channel ID

ABCKidsSydney.au

### Notes (optional)

I wanna watch the new bluey episodes 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19869
